controllers/apps/operations/ops_progress_util.go

```diff
@@ -570,3 +528,32 @@ func syncProgressToOpsRequest(
 	}
 	return nil
 }
+
+func getPodSetForComponent(
```

Comment:
If the rule which genereates Pod names changes, this function should be changed accordingly.

It's better this function is put inside in the module that is responsible to generate Pods, e.g. Components or InstanceSet.